src/Ucommerce.Avenue.Umbraco/Api/AvenueClothingApiBasketController.cs

```diff
@@ -98,7 +91,7 @@ public IHttpActionResult GetBasket()
                 var product = CatalogLibrary.GetProduct(line.Sku);
                 var url = UrlService.GetUrl(CatalogContext.CurrentCatalog, new[] {product});
                 var imageUrl = product.PrimaryImageUrl;
-                var lineTotal = new Money(line.Total.Value, currency);
+                var lineTotal = new ApiMoney(line.Total.Value, CatalogContext.CurrentPriceGroup.CurrencyISOCode);
```

Comment:
`var currencyISOCode = CatalogContext.CurrentPriceGroup.CurrencyISOCode` and use it in all the cases.

Reply:
Agree, 5 use cases replaced